wordpress-mobile/WordPress-iOS
# Schedule post in the Nudges view

Fixes #13784

### To test

1. Create a new post
2. Tap "Publish"
3. Tap "Publish" again to schedule your post
4. Tap "Publish Now"
5. Confirm that the post was scheduled

Please, test using iPhone and iPad.
